### ISSUE TITLE
kallisto stranded file has extra column

### ISSUE THREAD

**Issue by jaclyn-taroni** [AlexsLemonade/OpenPBTA-analysis#474]
Context is the loss of feature names here: https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/470/commits/ac8576abb5e8882a35b38ea5d573c58e6c11c6a6#diff-557d92e4b6e60334985b018096e31fab

#### What data file(s) does this issue pertain to?

kallisto stranded RDS file `pbta-gene-expression-kallisto.stranded.rds`

#### What release are you using?

`release-v13-20200116`

Double checking that I'm looking at the correct file: 

```sh
$ md5sum data/release-v13-20200116/pbta-gene-expression-kallisto.stranded.rds 
e8d9f288e87ae0f3f7a45806822d9ea2  data/release-v13-20200116/pbta-gene-expression-kallisto.stranded.rds
```
from `data/release-v13-20200116/md5sum.txt`

```
e8d9f288e87ae0f3f7a45806822d9ea2  pbta-gene-expression-kallisto.stranded.rds
```

#### Put your question or report your issue here.

There's an extra first column that contains what looks like row indices:

```r
> kallisto_stranded <- readr::read_rds("data/pbta-gene-expression-kallisto.stranded.rds")
> kallisto_stranded[1:5, 1:5]
  X                 transcript_id                   gene_id BS_S2TA8R29 BS_JMHG7GCN
1 0    ENST00000000233.9_ARF5-201   ENSG00000004059.10_ARF5   15.695300   17.966400
2 1    ENST00000000412.7_M6PR-201    ENSG00000003056.7_M6PR    7.616950   11.075800
3 2  ENST00000000442.10_ESRRA-201  ENSG00000173153.13_ESRRA    1.542180    1.384210
4 3   ENST00000001008.5_FKBP4-201   ENSG00000004478.7_FKBP4    4.297440    6.415390
5 4 ENST00000001146.6_CYP26B1-201 ENSG00000003137.8_CYP26B1    0.359054    0.563337
```

**Comment by tkoganti** [comment]
I will be working on this after the subtyping ticket

**Comment by jharenza** [comment]
@jaclyn-taroni and @tkoganti I just fixed this and will add to #432

**Comment by jharenza** [comment]
closed via #507
